### DISCUSSION TITLE
Feedback to build instructions

### DISCUSSION THREAD

**Discussion by mariuszhermansdorfer** [cadop/dhart#37] [General]
@cadop, as requested I'm opening a separate thread to keep track of the build steps.

I'm on VS 2022 and it doesn't come with .NET 4.5 anymore. These are the options users are presented with:
![image](https://user-images.githubusercontent.com/49192999/204157793-a6b20084-641c-4129-8ad7-48b7cd045e0a.png)

I bumped the Framework to 4.8 and VS version to 2022. 
This resulted in a clean build in the x64-Release-CSharp mode.

I submitted these as a PR to the Rhino branch

What I'm still having issues with is the git setup. I can't switch branches or push anything from VS. The git desktop client, however, allows me to do both.  For now, I will just use this workaround.

**Comment by cadop** [comment]
Thanks for this, did you install everything else in the instructions, or you went with a minimal approach?  I am wondering how many things I can remove from the list :)

**Reply by mariuszhermansdorfer** [reply]
I went with a minimal approach - only the default VS 2022 installation, c++ & c# development tools and I had cmake and git installed before.

**Comment by cadop** [comment]
I think github desktop is good, I usually use that when I want to make sure I'm not pushing files that I don't want to.

**Reply by mariuszhermansdorfer** [reply]
It's the first time for me that the built-in git tools for VS behave incorrectly. I find it convenient to switch between branches and push commits from the VS workspace. But, as mentioned before, git for desktop works fine.
Maybe this could be something to add to the build instructions as a gotcha.

**Reply by cadop** [reply]
Intersting, I guess it would help to open an issue for this because I don't see how the two things are connected.

**Comment by cadop** [comment]
I have updated the `build.md` to include some FAQ and instructions mentioning the .net framework version.  Thanks for bringing this up.

https://github.com/cadop/dhart/pull/43
